src/common/httpClient/httpClient.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export const httpClient = axios.create({
+  baseURL: `https://quizapi.io/api/`,
```

Comment:
Chyba nie to API 😉

Reply:
Poprawione. podłączyłem frankfurter api.